UnityMcpBridge/Editor/Windows/ManualConfigEditorWindow.cs

```diff
@@ -115,6 +121,14 @@ protected virtual void OnGUI()
                     instructionStyle
                 );
             }
+            else if (mcpClient?.mcpType == McpTypes.Warp)
```

Comment:
We don't have Warp as a client as yet. why is this code being added to `main`? Maybe you want to merge to #351 instead